src/Actions.js

```diff
@@ -35,7 +35,7 @@ export function create({ name, map }) {
   function action(value) {
     let err = null;
     try {
-      value = map(value);
+      value = map.bind(this)(value);
     } catch(e) {
```

Comment:
The context of the `create` function is already the stamp, but even with the explicit passing of `instance`, it won't change the context of the `map` function (which is `undefined`). Or am I misunderstanding what you're suggesting?